operations/dynamic-templates/isolation-segment.yml

```diff
@@ -0,0 +1,45 @@
+--- # cleanup-params-passed
+iso_group: (( prune ))
```

Comment:
Ah yes, I wanted to clear out tmp created maps from manifest.
tmp map of name ${iso_group_name} is created so that it can be referenced in dynamic template via (( grab )).
I didn't yet solved how to clean it after spruce completes evaluation.

Reply:
https://play.spruce.cf/#d51c9d6bf1a17dfba72d9e405cafa971

Reply:
Sorted it out by grabbing values directly vs tmp yml map.